cfme/infrastructure/virtual_machines.py

```diff
@@ -32,16 +32,16 @@
 from cfme.infrastructure.provider.rhevm import RHEVMProvider
 from cfme.web_ui import (
     CheckboxTree, Form, InfoBlock, Region, Tree, fill, flash, form_buttons,
-    match_location, Table, toolbar, Calendar, Select, Input, CheckboxTable,
+    match_location, toolbar, Calendar, Select, Input, CheckboxTable,
     summary_title, BootstrapTreeview, AngularSelect)
+from cfme.web_ui import Table as webui_Table
```

Comment:
you can give Table other name in main web_ui import statement.
Also please give better name to web_ui Table. something like OldUITable.

Reply:
changed.